### DISCUSSION TITLE
React renderer + NodeFormattingExtension

### DISCUSSION THREAD

**Discussion by adyz** [remirror/remirror#1653] [General]
Hello! 

Fist I want to say that I think this is an awesome project! 

---

I've setup my editor with some basic wysiwyg functionalities like, Bold, Italic and Alignment. 
For alignment I've used NodeFormattingExtension
When I add this extenion, the renderer gives me this React error 

```mdx
The `style` prop expects a mapping from style properties to values, not a string. For example, style={{marginRight: spacing + 'em'}} when using JSX.
```

I've tried using `skipUnknownMarks={true}` and `kipUnknownTypes={true}`, but I still get the error.

I haven't found any documentation for renderer where I have used extensions (or event custom extensions)

**Comment by adyz** [comment]
Here is the example with the error: https://codesandbox.io/s/remirror-preview-bug-ti85f5
Here is the example, but with the extension commented: https://codesandbox.io/s/remirror-preview-bug-commented-0vr1yg

**Comment by hawc** [comment]
This issue is a duplicate of [#1285](https://github.com/remirror/remirror/issues/1285)
